imageregistry/v1/stable.config.testsuite.yaml

```diff
@@ -16,3 +16,72 @@ tests:
         logLevel: Normal
         operatorLogLevel: Normal
         replicas: 1
+  - name: Should be able to configure internal storage network access for Azure
+    initial: |
+      apiVersion: imageregistry.operator.openshift.io/v1
+      kind: Config
+      spec:
+        replicas: 1
+    expected: |
+      apiVersion: imageregistry.operator.openshift.io/v1
+      kind: Config
+      spec:
+        logLevel: Normal
+        operatorLogLevel: Normal
+        replicas: 1
+        storage:
+          azure:
+            type: Internal
```

Comment:
Missing type in the initial configuration defaults to External. This should fail.